lib/facility.rb

```diff
@@ -1,14 +1,89 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
-
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
-    @services = []
+  attr_reader :name, :address, :phone, :services, :registration_date, :vehicle_details, :license_data
+  attr_accessor :registration_date, :name, :address, :phone
```

Comment:
Some of these aren't attributes in this class so we don't need them.
```suggestion
  attr_reader :name, :address, :phone, :services
  attr_accessor 
```